mini-modeline.el

```diff
@@ -296,6 +296,14 @@ BODY will be supplied with orig-func and args."
 
 (defsubst mini-modeline--exit-minibuffer ()
   "`minibuffer-exit-hook' of mini-modeline."
+  (mapc
+   (lambda (buf)
+     (with-current-buffer buf
+       (when (and mini-modeline-enhance-visual
+                  (or (minibufferp buf)
+                      (string-prefix-p " *Echo Area" (buffer-name))))
+         (mini-modeline--set-buffer-face))))
+   (buffer-list))
```

Comment:
I think this can be simplified as
``` elisp
  (when mini-modeline-enhance-visual
    (with-current-buffer mini-modeline--minibuffer
      (mini-modeline--set-buffer-face)))
```

Reply:
You can sync to latest HEAD, I've defined new variable `mini-modeline-minibuffer` to help with this case in 08bd65d

Reply:
Done, thank you!:)